app/scripts/controllers/landing.js

```diff
@@ -44,6 +45,11 @@ angular.module('confRegistrationWebApp')
         };
         $scope.searchComplete = val;
         $scope.searchResults = response.data;
+
+        analytics.digitalData.searchFilter = '';
```

Comment:
@adammeyer @OzzieOrca this was used on the Give site to capture the name of the search filters if someone clicks on them. It tells us if people interact with the filters to help them find what they are looking for. For ERT it could be used to capture the names of the "locations" or "date" options when searching but right now we don't have to worry about that. Let's leave it in the data layer though.